zulipterminal/ui_tools/boxes.py

```diff
@@ -1726,7 +1726,7 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
             self.panel_view.keypress(size, "esc")
         elif is_command_key("ENTER", key) and not self.panel_view.empty_search:
             self.panel_view.view.controller.exit_editor_mode()
-            self.set_caption([("filter_results", " Search Results"), " "])
+            self.set_caption([("filter_results", " Search Results\n"), " > "])
```

Comment:
Let's wait for everybody's views on this in the stream.

Reply:
Agreed - also if this is separate to other changes that are less controversial then having it last make it easier to merge the others first.